tensorflow/lite/experimental/micro/memory_planner/greedy_memory_planner.h

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 #ifndef TENSORFLOW_LITE_EXPERIMENTAL_MICRO_MEMORY_PLANNER_GREEDY_MEMORY_PLANNER_H_
 #define TENSORFLOW_LITE_EXPERIMENTAL_MICRO_MEMORY_PLANNER_GREEDY_MEMORY_PLANNER_H_
 
+#include "tensorflow/lite/experimental/micro/compatibility.h"
```

Comment:
`In file included from third_party/tensorflow/lite/experimental/micro/memory_planner/greedy_memory_planner.cc:16:
./third_party/tensorflow/lite/experimental/micro/memory_planner/greedy_memory_planner.h:19:10: fatal error: 'third_party/tensorflow/lite/experimental/micro/compatibility.h' file not found
#include "third_party/tensorflow/lite/experimental/micro/compatibility.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.`

can you please check above error which is reported internally.

Reply:
Hi, I don't know how to reproduce the issue. When I make test the tensorflow lite code it compiles on my machine.

Reply:
@petewarden any thoughts how to fix this error ? Thank you

Reply:
It looks like you're run into a BUILD dependency issue.  When we run into an issue like this, the right thing to do is split compatibility.h into it's own BUILD target in micro/BUILD, and add //third_party/tensorflow/lite/micro:compatibility to micro_framework's deps and to greedy_memory_planner's deps.  Unfortunately you'll also have to fix up your file paths since we recently moved micro out of experimental - so move your files to tensorflow/lite/micro instead of tensorflow/lite/experimental/micro.  Additionally, all your includes will have to change to remove experimental/.  If you would prefer, I can make the necessary changes internally and close this PR.

Reply:
Please make the necessary changes and close the PR.